exchanges/apis/kucoin.py

```diff
@@ -1,10 +1,21 @@
+from typing import Dict
+import base64
+import hashlib
+import hmac
+import json
+import time
+
+from loguru import logger
+from ratelimiter import RateLimiter
+
 from .base import BaseExchangeApi, ExchangeApiException
 
 
 class KuCoinApi(BaseExchangeApi):
     api_prefix = "api"
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, passphrase=None, *args, **kwargs):
```

Comment:
Please be explicit about the args when they are known (as in this case)

Reply:
what do you mean? I'm trying to allow an invocation like:

`KuCoinApi(passphrase='pass', secret='secret')`

where in this case `secret` is passed to the `super().__init__(*args, **kwargs)`

unless you are referring to something else?

Reply:
The set of possible args is known (from the base class's __init__), so I
don't think we want to use *args, **kwargs here. You can just specify all
of them, so users of the class don't have to go looking. Also,
passphrase=None -- wouldn't it be an error to init this class without a
passphrase? If so, it shouldn't have a default.